src/node-fs.js

```diff
@@ -18,7 +19,7 @@ module.exports = {
       return http.get(path, encoding)
     }
     if (path.startsWith('file://')) {
-      return fs.readFileSync(path.substr('file://'.length), encoding)
+      return fs.readFileSync(url.fileURLToPath(path), encoding)
```

Comment:
:+1:

Reply:
Yes, the following test cases fail when running in Windows (not Linux):

- should throw an error for unexisting file referenced with "file" protocol
- should return diagramText with inlined local file referenced with "file" protocol and absolute path

Btw: I get new test failures since my last commit and even on master. Do you know why? Perhaps kroki-server changes?:

- should inline (via page attr) an SVG image with built-in allow-uri-read (available in Asciidoctor.js 2+)
- should inline (via block attr) an SVG image with built-in allow-uri-read (available in Asciidoctor.js 2+)
- should inline (via block attr overriding page attr) an SVG image with built-in allow-uri-read (available in Asciidoctor.js 2+)

Reply:
Thanks for the clarification.

> Btw: I get new test failures since my last commit and even on master. Do you know why? Perhaps kroki-server changes?:

It should be fixed in https://github.com/Mogztter/asciidoctor-kroki/commit/a6bc1853eacc2ba9846028c6fc951b1557f9c4be (pushed yesterday on master).
Did you fetch + rebase on master ?

Reply:
Thanks, that solved the problem.